app/views/issues/nodes/_evidence.html.erb

```diff
@@ -33,7 +33,7 @@
               <h3>
                 Instance #<%= i+1 %> for <%= link_to node.label, project_node_path(node.project, node) %> -
                 <span class="actions">
-                  <%= link_to edit_project_node_evidence_path(node.project, node, instance), class: 'action-link' do %>
+                  <%= link_to edit_project_node_evidence_path(node.project, node, instance, back_to: :issue), class: 'action-link' do %>
```

Comment:
it seems this change is no longer needed?

Reply:
this `back_to` param solves the evidence edit from issue case, when we have > 1 evidence for the issue.
When we have just 1 evidence the param was already there https://github.com/dradis/dradis-ce/pull/396/files#diff-05c2eb1fbc4727068bb8269ff00dfbe3R8